data/human/news.txt

```diff
@@ -1217,3 +1244,30 @@ news "graffiti on spaceport wall"
 			"You think piracy is bad and want to stop us? You and what army?"
 			"Messing with the Syndicate, Does That Sound Like Fun to You?"
 			"83 percent of all statistics are just made up."
+			"Ask not what your planet can do for you; ask what you can do for your planet. - based on John F. Kennedy"
+			"Fair is foul, and foul is fair - Shakespeare, Macbeth"
+			"For want of a shoe the crewmember was lost. / For want of a crewmember the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."
```

Comment:
```suggestion
			"For want of a shoe, the crewmember was lost. / For want of a crewmember, the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."
```
OR
```suggestion
			"For want of a shoe the crewmember was lost. / For want of a crewmember the ship was lost. / For want of a ship the fleet was lost. / For want of a fleet the battle was lost. And all for the want of a shoe."
```
Commas could go either way, but I think being consistent with one way or the other is a good idea.